pkg/dashboards/dashboard.go

```diff
@@ -99,6 +99,9 @@ const getDashboardEntityQuery = `query ($guid: EntityGuid!) {
             value
           }
           name
+		  options {
+				ignoreTimeRange
+			}
```

Comment:
```suggestion
          options {
            ignoreTimeRange
          }
```

just a tinyformatting fix